chalk-ir/src/lib.rs

```diff
@@ -1,13 +1,15 @@
 #![deny(rust_2018_idioms)]
 
+#[macro_use]
+extern crate lazy_static;
```

Comment:
Can we directly import `lazy_static::lazy_static`, or is it still idiomatic to `macro_use` it (compared to when we directly import the debug macros)?

Reply:
Good catch! I'll change this.